tools/cloudflow-it/Makefile

```diff
@@ -59,6 +54,12 @@ prepare-cluster:
 		--version "${version}" \
 		--set kafkaClusters.default.bootstrapServers=cloudflow-strimzi-kafka-bootstrap.cloudflow:9092 \
 		--namespace cloudflow)
+	@echo '****** Installing Spark Operator'
+	helm repo add incubator https://charts.helm.sh/incubator | true
+	helm repo update
+	helm upgrade -i spark-operator incubator/sparkoperator --values="spark-values.yaml"  --version "0.6.7" --namespace cloudflow
```

Comment:
Should this not be latest (or some stable version close to latest) as well, since all the tests work with latest operator?

Reply:
( I'm talking about `--version`, where later on you set operatorVersion to 'latest')

Reply:
omitted the helm chart version as discussed.